subsys/bluetooth/controller/ll_sw/lll_adv.h

```diff
@@ -127,12 +131,26 @@ struct lll_adv_aux {
 	 */
 	uint16_t data_chan_counter;
 
+	uint16_t data_chan_id;
+	struct {
+		uint8_t data_chan_map[PDU_CHANNEL_MAP_SIZE];
+		uint8_t data_chan_count:6;
+	} chm[DOUBLE_BUFFER_SIZE];
+	uint8_t  chm_first;
+	uint8_t  chm_last;
+
```

Comment:
Two reasons:
1 - to align with lll_adv_sync/lll_adv_iso which has the channel map in the LLL structure
2 - since the channel map update and channel index calculation is now done in prepare, it made sense to have it in the LLL structure

As you mentioned in the meeting, the channel map update should be moved though

Reply:
Moved this back to ull and made the discussed changes to channel map updates